pallets/handles/src/handles-utils/src/converter.rs

```diff
@@ -84,7 +84,7 @@ pub fn split_display_name(display_name_str: &str) -> Option<(String, HandleSuffi
 	let parts: Vec<&str> = display_name_str.split(HANDLE_DELIMITER).collect();
 	let base_handle_str = parts[0].to_string();
 	if parts.len() != 2 {
-		return None
+		return None;
```

Comment:
My formatter really wants to have `;` at the end of the line when it is `return`